.gitignore

```diff
@@ -52,6 +52,7 @@ stamp-h1
 *~
 *.cache
 *#*
+.vscode
```

Comment:
What's a .vscode file?

Reply:
Visual Studio Code. That should have been gitignored. I fix it.